### ISSUE TITLE
"Show Desktop" via clicking far-right side on the taskbar doesn't work

### ISSUE THREAD

**Issue by Velgus** [LuisThiamNye/ButteryTaskbar2#15]
When Buttery Taskbar 2 is enabled, the Show Desktop feature via clicking the far-right side of the taskbar doesn't work. The hotkey (Win+D) still works fine.

**Comment by LuisThiamNye** [comment]
I've taken a little look at this and it seems to me that the problem is that the Show Desktop button doesn't work when the Start menu is open, at least with taskbar auto-hide enabled. This issue happens even when Buttery Taskbar is not running.

It gets better though, because if you open the Start menu then click an empty space on the taskbar, not only does it close the Start menu, but it also moves the taskbar off-screen (with auto-hide) because Microsoft is a super serious company hiring super serious programmers who definitely know what they are doing and they care about the customer experience very very much.

In summary, this is Microsoft's issue and I don't know how to fix it.
